catalog/message/dataset.json

```diff
@@ -1,44 +1,49 @@
 {
-  "@context":  "https://w3id.org/dspace/1/0/context.json",
+  "@context": "https://w3id.org/dspace/v0.8/context.json",
   "@id": "urn:uuid:3afeadd8-ed2d-569e-d634-8394a8836d57",
   "@type": "dcat:Dataset",
   "dct:title": "Traffic Data",
-  "dct:description": "Traffic data sample extract",
-  "dcat:keyword": [
-    "traffic"
+  "dct:description": [
+    {
+      "@value": "Traffic data sample extract",
+      "@language": "en"
+    }
   ],
-  "odrl:hasPolicy": [
+  "dcat:keyword": [
     {
-      "@context": {
-        "@vocab": "https://www.w3.org/TR/odrl-model/"
-      },
-      "@id": "urn:uuid:2828282:3dd1add8-4d2d-569e-d634-8394a8836a88",
-      "permission": [
-        {
-          "action": "use",
-          "constraint": [
-            {
-              "leftOperand": {
-                "@value": "spatial"
-              },
-              "rightOperand": {
-                "@value": "EU"
-              },
-              "operator": "EQ"
-            }
-          ],
-          "duty": []
-        }
-      ],
-      "prohibition": [],
-      "obligation": []
+      "@value": "traffic",
+      "@language": "en"
     }
   ],
+  "odrl:hasPolicy": {
+    "@type": "odrl:Offer",
+    "@id": "urn:uuid:2828282:3dd1add8-4d2d-569e-d634-8394a8836a88",
+    "odrl:permission": [
+      {
+        "odrl:action": "odrl:use",
+        "odrl:constraint": [
+          {
+            "odrl:leftOperand": "odrl:spatial",
+            "odrl:rightOperand": {
+              "@id": "_:EU"
+            },
+            "odrl:operator": "odrl:eq"
+          }
+        ]
+      }
+    ]
+  },
   "dcat:distribution": [
     {
       "@type": "dcat:Distribution",
-      "dct:format": "dspace:s3+push",
-      "dcat:accessService": "urn:uuid:4aa2dcc8-4d2d-569e-d634-8394a8834d77"
+      "dct:format": {
+        "@id": "dspace:s3+push"
+      },
+      "dcat:accessService": [
+        {
+          "@id": "urn:uuid:4aa2dcc8-4d2d-569e-d634-8394a8834d77"
```

Comment:
Given another comment I added, the DataService should also include a `@type`, otherwise it should be a reference